### ISSUE TITLE
Remember settings for dataset creation and training

### ISSUE THREAD

**Issue by RenNagasaki** [erew123/alltalk_tts#203]
Please, do me the favour of saving these settings.
I'm having to train a shitload of voices, and having to refill all these fields each time, is killing me. 🤣

![image](https://github.com/erew123/alltalk_tts/assets/10595885/fdb82db8-9eec-4c0e-9e60-835d568ecdda)

**Comment by erew123** [comment]
Ill see what I can do on the next version.

If for now you want to fix the values in finetune.py:

Learning rate is line 1447 `value=5e-6,`

and the others are lines 1142 to 1165:

```
    parser.add_argument(
        "--num_epochs",
        type=int,
        help="Number of epochs to train. Default: 10",
        default=10,
    )
    parser.add_argument(
        "--batch_size",
        type=int,
        help="Batch size. Default: 4",
        default=4,
    )
    parser.add_argument(
        "--grad_acumm",
        type=int,
        help="Grad accumulation steps. Default: 1",
        default=1,
    )
    parser.add_argument(
        "--max_audio_length",
        type=int,
        help="Max permitted audio size in seconds. Default: 11",
        default=11,
    )
```

Thanks

**Comment by RenNagasaki** [comment]
Love you! Thanks!
